pymc3/distributions/continuous.py

```diff
@@ -2642,8 +2655,13 @@ def logcdf(self, value):
         """
         alpha = self.alpha
         beta = self.beta
+        # To avoid issue with #4340
```

Comment:
Again better explain the actual issue here rather than point elsewhere. And better still to put this information right in the commit message.